src/omega_edit/client.ts

```diff
@@ -26,6 +26,7 @@ import { DataEditWebView } from './dataEditWebView'
 import { initOmegaEditClient } from './utils'
 
 export let serverPort: number = 9000
+export let serverPortLock: boolean = false
```

Comment:
Instead of introducing another variable, we can use the serverPort above.  Just make it either 0, or 'undefined'.  The first instance will set it to a positive number and the second instance can check if the port number is positive, which is should be and lock the user out from creating a second DE.